src/app/ui/dataselector.js

```diff
@@ -0,0 +1,48 @@
+'use client';
+
+import { useState, useContext, useEffect } from 'react';
+import Calendar from 'react-calendar';
+
+import { TokenContext } from "@/app/ui/login.js";
+
+import { get } from '@/app/lib/evduty_api.js';
+
+import styles from "@/app/ui/page.module.css";
+
+function TerminalSelector() {
+    return (
+        <div className={styles.terminalselector}>
+            <select name='terminals' id='terminals' multiple>
+                <option value='terminal1'>Terminal 1</option>
+                <option value='terminal2'>Terminal 2</option>
+                <option value='terminal3'>Terminal 3</option>
+            </select>
+        </div>
+    )
+}
+
+function AveragingSelector() {
+    return (
+        <div className={styles.averagingselector}>
+            <input type="radio" id="averaging" name="averaging" value="Monthly" />
```

Comment:
`id`s must be unique across the whole document, so you can do something like `averaging-monthly` (and then use the same value in `htmlFor`.